### PR TITLE
Update dockerfile to use serve instead of http.server

### PR DESCRIPTION
**Pull request by hatched**
## Done

This change allows reviewers to refresh the application at any route and have the correct assets served for that path instead of having it 404 on all paths but `/`.

## QA

- Visit the demo link and navigate around the application.
- Refresh at various points and it should load the app to those pages.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,29 +8,10 @@ ADD package.json .
 ADD yarn.lock .
 RUN --mount=type=cache,target=/usr/local/share/.cache/yarn yarn install
 
-
-# Build stage: Run "yarn run build-js"
-# ===
 FROM yarn-dependencies AS build-js
```
Comment (nottrobin) [inline Dockerfile:11]: It's pointless splitting this into 2 stages if the second stage is just going to inherit from the first

Comment (nottrobin) [reply Dockerfile:11]: You also don't need to give your final stage a name. It's not going to be used by anything.

File: Dockerfile [hunk 2]
```diff
@@ -8,29 +8,10 @@ ADD package.json .
...
-# Import code, build assets and mirror list
-ADD . .
-RUN rm -rf package.json yarn.lock .babelrc webpack.config.js
-COPY --from=build-js /srv/build .
+RUN yarn global add serve
 
 # Setup commands to run server
-ENTRYPOINT ["python3"]
-CMD ["-m", "http.server", "80"]
+ENTRYPOINT ["serve"]
```
Comment (nottrobin) [inline Dockerfile:17]: As João said, you should serve on port 80 if you can.

Comment (hatched) [reply Dockerfile]: Updated this to serve on port 80

File: Dockerfile (hunk 1 above)
Comment (hatched) [reply Dockerfile]: Good call, updated.

**Review by barrymcgee — APPROVED** [review]

**Pull request merged**